src/components/Foods.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import foodSeed from '../foods.json'
+import FoodBox from './FoodBox'
+import AddFood from './AddFood'
+import TodayFood from './TodayFood'
+
+
```

Comment:
remember to check spacing on your projects and set it to 2. Also, take a look at the auto-indentation feature of vs code with prettier to see how the code should be structured.